pkg/handlerutil/handlerutil.go

```diff
@@ -78,6 +79,8 @@ func ParseAndGetChart(req *http.Request, cu chartUtils.Resolver, requireV1Suppor
 	if err != nil {
 		return nil, nil, err
 	}
+	chartDetails.UserToken = auth.ExtractToken(req.Header.Get("Authorization"))
```

Comment:
I would rather use a different param for the UserToken for two reasons: that way we are not returning a partially empty struct from `ParseDetails` and also because we avoid having non-related chart info y the chart details.

Reply:
Yep, makes sense. I'll separate `UserToken` out of `ChartDetails` tomorrow. Thanks.